server/build.gradle.kts

```diff
@@ -114,24 +109,34 @@ val TachideskRevision = Runtime
 
         }
 
+launch4j { //used for windows
+    mainClassName = MainClass
+    bundledJrePath = "jre"
+    bundledJre64Bit = true
```

Comment:
64-bit or 32-bit?

Reply:
That config I just moved here because the tasks are now in the tasks block, I didnt edit any of your values other then changing the main class variable name